### PR TITLE
Fix compilation with clang 13

### PR DESCRIPTION
**Pull request by h0tc0d3**
Fix this issue with clang 13 https://github.com/AOMediaCodec/libavif/issues/782

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -129,6 +129,13 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
         -Wno-sign-conversion
         -Wno-error=c11-extensions
     )
+    # Some headers such as glib-2.0 use identifier names that start with '_' followed by a capital letter.
+    # The C standard reserves names beginning with an underscore and various other combinations. 
+    # ISO/IEC 9899:1999 (aka C99 standard) §7.1.3 Reserved identifiers
```
Comment (wantehchang) [inline CMakeLists.txt:134]: Could you please change the '§' character to "Section" or "Sec."? We are trying to use only ASCII characters in the source code.

**Review by wantehchang — CHANGES_REQUESTED** [review]
My apologies for the late review. I am out of the office.

File: CMakeLists.txt (hunk 1 above)
Comment (h0tc0d3) [reply CMakeLists.txt]: Yep. I removed '§' character. 
I think it will be better.

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks!

**Comment by wantehchang** [comment]
I am very sorry that I forgot to merge this pull request. I will do that now.

**Pull request merged**